Automattic/woocommerce-payments
# Reporting: Add preset date selector to the Payment activity widget

Fixes #8734 

#### Changes proposed in this Pull Request
 * Add a dropdown of preset date ranges 
 * Selecting a date range should update the label 
 * It should also update the data in the payment activity widget matching the selected date range

#### Not included in this PR
* The All-time date range will be covered in a separate issue since it needs exploration on the best way to get the start date. Currently, using the account creation date available within `wcpaySettings.accountStatus` appears to be a good contender - #8929 
*  Issue #8928 - The Inline select item used requires a change in the button label format to make it match with the design. 
 
#### Testing instructions
* Make sure the feature flag - `_wcpay_feature_payment_overview_widget` is enabled
* Check out this branch on your local test site
* Browse to `Payment > Overview` and see that the date selector renders in the top right corner with default option of `Last 7 days`

<a href="https://github.com/Automattic/woocommerce-payments/assets/4162931/b4653db9-b1e4-40cd-9b99-60fe194457a8"><img src="https://github.com/Automattic/woocommerce-payments/assets/4162931/b4653db9-b1e4-40cd-9b99-60fe194457a8" width="600px" /></a>

* Select various options of the dropdown and see how the data changes to match the range


#### Additional 

* The PR uses the newly created `InlineLabelSelect` component. 🙌  - @Jinksi .

-------------------


**Post merge**
